### ISSUE TITLE
App-Veränderungen automatisch erkennen

### ISSUE THREAD

**Issue by patrickhilker** [mittwald/feature-requests#55]
**Welches Problem möchtest du lösen? Wann tritt es auf?**
Verschiedene Apps, beispielsweise WordPress, ist in der Lage, sich selbst zu aktualisieren. In diesem Fall wird im mStudio weiterhin die alte Version für diese App angezeigt. Dazu kommt, dass sogar ein mögliches Update angezeigt wird. Das liegt daran, dass diese "externe" Versionsänderung nicht vom mStudio "bemerkt" wird.

**Welche Lösungsideen hast du?**
Die Projekte werden automatisch regelmäßig auf geänderte App-Versionen, neue Apps oder gelöschte Apps analysiert. Diese Änderungen werden dann im mStudio dargestellt.

**Hast du zusätzliche Informationen (wie z.B. Screenshots)?**

**Comment by patrickhilker** [comment]
Eine erste Variante möchten wir demnächst testen. Gerne schalten wir ausgewählte Projekte für einen ersten Testlauf frei. Gerne hier mit einer Projek-ID (p-xxxxxx) melden.

**Comment by vogelor** [comment]
p-kn66tj

**Comment by 16NEUN** [comment]
p-zq9r7h

**Comment by rteitge** [comment]
p-aht2g8

**Comment by g-nebel** [comment]
Ja, dieses Feature wäre sehr praktisch vor allem im Zusammenhang mit einer TYPO3 Composer Installation.
Auch das Triggern der Neuerkennung über die API wäre eine Variante.

Projek-ID: p-d3h21f

**Comment by jankal** [comment]
Wir haben das gleiche Problem mit Nextcloud. p-ll0sbj - das mStudio weiß nicht, dass ich die Nextcloud inzwischen manuell über den Updater angehoben habe und bietet mir das Update weiterhin an.
